### PR TITLE
Update .env.local in README

### PR DESCRIPTION
**Pull request by kanterov**
Now when we support S3, use settings compatible with Flyte sandbox

Fixes https://github.com/lyft/flyte/issues/332

### CODE REVIEW

**Comment by kanterov** [comment]
@kumare3 we have added support for S3. Can you please try it out, or find somebody else who can do that?

**Comment by wild-endeavor** [comment]
This is the issue for this PR right? https://github.com/lyft/flyte/issues/332. Mind adding it to the description @kanterov ?

**Comment by kumare3** [comment]
So you already merged the s3 client in. We will try it in some time

**Comment by kumare3** [comment]
ok let me try it out now

File: README.md [hunk 1]
```diff
@@ -40,8 +40,11 @@ We don't publish artifacts yet, but you can build examples yourself.
 Create `.env.local` with:
 
 ```bash
-FLYTE_PLATFORM_URL=flyte.local:81
-FLYTE_STAGING_LOCATION=gs://yourbucket
+FLYTE_PLATFORM_URL=localhost:30081
+FLYTE_AWS_ENDPOINT=http://localhost:30084
+FLYTE_AWS_ACCESS_KEY_ID=minio
+FLYTE_AWS_SECRET_ACCESS_KEY=miniostorage
+FLYTE_STAGING_LOCATION=s3://flyteorg
```
Comment (sbrunk) [inline README.md:47]: The bucket name has changed in more recent sandbox environments:
```suggestion
FLYTE_STAGING_LOCATION=s3://my-s3-bucket
```

**Review by sbrunk — APPROVED** [review]
I just tried that successfully with a new sandbox. So LGTM with the suggested bucket name change.

**Pull request merged**